library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -466,6 +466,22 @@ public void onItemMoved(@NonNull Group group, int fromPosition, int toPosition)
         notifyItemMoved(groupAdapterPosition + fromPosition, groupAdapterPosition + toPosition);
     }
 
+    @Override
+    public void onViewAttachedToWindow(@NonNull VH holder) {
```

Comment:
this looks great, but could you please move these functions to be just under the `onFailedToRecycleView` one so that we can group the `RecyclerView.Adapter` overloads in one place

Reply:
That makes sense. They should be in the correct position now.